<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(162, 109, 159)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_quant">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QUine Edit {
border: 2px sold rgb(37, 39, 48);
border-radius: 20px; 
color: #FFF; 
padding-left: 20px; 
padding right: 20px; 
background-color: rgb(34, 36, 44);
}
QUineEdit:hover {
border: 2px solid rgb(48, 50, 62);
}
QUineEdit:focus{
border: 2px solid rgb(85, 170, 255); 
background -color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Quantité</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modifier">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ref">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QUine Edit {
border: 2px sold rgb(37, 39, 48);
border-radius: 20px; 
color: #FFF; 
padding-left: 20px; 
padding right: 20px; 
background-color: rgb(34, 36, 44);
}
QUineEdit:hover {
border: 2px solid rgb(48, 50, 62);
}
QUineEdit:focus{
border: 2px solid rgb(85, 170, 255); 
background -color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Reference </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_recherche">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>351</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QUine Edit {
border: 2px sold rgb(37, 39, 48);
border-radius: 20px; 
color: #FFF; 
padding-left: 20px; 
padding right: 20px; 
background-color: rgb(34, 36, 44);
}
QUineEdit:hover {
border: 2px solid rgb(48, 50, 62);
}
QUineEdit:focus{
border: 2px solid rgb(85, 170, 255); 
background -color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Rechercher un produit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-image: url(C:/Users/medzi/Downloads/LOGO Fashionhub.jpg);
background-repeat: no-repeat;
background-position: center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Qualité</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_couleur">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QUine Edit {
border: 2px sold rgb(37, 39, 48);
border-radius: 20px; 
color: #FFF; 
padding-left: 20px; 
padding right: 20px; 
background-color: rgb(34, 36, 44);
}
QUineEdit:hover {
border: 2px solid rgb(48, 50, 62);
}
QUineEdit:focus{
border: 2px solid rgb(85, 170, 255); 
background -color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="placeholderText">
     <string>Couleur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_supprimer">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnbtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Accueil</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 255)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_afficher">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QTableView" name="tab_produit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  width: auto;
    height: auto;
    overflow: hidden;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_taille">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QUine Edit {
border: 2px sold rgb(37, 39, 48);
border-radius: 20px; 
color: #FFF; 
padding-left: 20px; 
padding right: 20px; 
background-color: rgb(34, 36, 44);
}
QUineEdit:hover {
border: 2px solid rgb(48, 50, 62);
}
QUineEdit:focus{
border: 2px solid rgb(85, 170, 255); 
background -color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="placeholderText">
     <string>Taille</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prix">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QUine Edit {
border: 2px sold rgb(37, 39, 48);
border-radius: 20px; 
color: #FFF; 
padding-left: 20px; 
padding right: 20px; 
background-color: rgb(34, 36, 44);
}
QUineEdit:hover {
border: 2px solid rgb(48, 50, 62);
}
QUineEdit:focus{
border: 2px solid rgb(85, 170, 255); 
background -color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="placeholderText">
     <string>Prix</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mediocre">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>600</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Mediocre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouter">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="f">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>530</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Femme</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Catégorie</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="low">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="high">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="h">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Homme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_trier">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Tri par quantite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_trier_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Tri par prix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_recherche">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_trier_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Tri par qualite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_generer_pdf">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouterimage">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>240</y>
      <width>61</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>Ajout IMG </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Référence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Prix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Taille</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Couleur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Quantité</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_updateStatus">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(53, 53, 53) ;
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>update etagiére</string>
    </property>
   </widget>
   <zorder>le_quant</zorder>
   <zorder>pb_modifier</zorder>
   <zorder>le_ref</zorder>
   <zorder>le_recherche</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>le_couleur</zorder>
   <zorder>pb_supprimer</zorder>
   <zorder>pb_afficher</zorder>
   <zorder>le_taille</zorder>
   <zorder>le_prix</zorder>
   <zorder>mediocre</zorder>
   <zorder>pb_ajouter</zorder>
   <zorder>f</zorder>
   <zorder>label_4</zorder>
   <zorder>low</zorder>
   <zorder>high</zorder>
   <zorder>h</zorder>
   <zorder>pb_trier</zorder>
   <zorder>pb_trier_2</zorder>
   <zorder>pb_recherche</zorder>
   <zorder>pb_trier_3</zorder>
   <zorder>pb_generer_pdf</zorder>
   <zorder>pb_ajouterimage</zorder>
   <zorder>label_2</zorder>
   <zorder>returnbtn</zorder>
   <zorder>tab_produit</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>pb_updateStatus</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
